src/generators/mysqlgenerator.cpp

```diff
@@ -345,4 +345,15 @@ QString MySqlGenerator::createConditionalPhrase(const PhraseData *d) const
     return SqlGeneratorBase::createConditionalPhrase(d);
 }
 
+void MySqlGenerator::appendSkipTake(QString &sql, int skip, int take)
```

Comment:
What about the case of (skip > 0 ; take == 0) ?
In some case user may be skip some record from beginning without limitation to remaining data